### ISSUE TITLE
Save File automatically loading when un-suspending on PC

### ISSUE THREAD

**Issue by kevr** [Kahmul/SoulsSpeedruns-Save-Organizer#11]
I just ran into a serious issue with SaveOrganizer during my playthrough which reverted about three bosses.

When suspending and unsuspending my machine with a save selected, with hotkeys setup to load the save, SpeedSouls seems to have loaded a save even though I never loaded it manually or pressed the load hotkey combination. Perhaps this has something to do with un-suspension awaking the keyboard or something.

Anyway, I would highly recommend that people close this program if they are going to suspend their Windows sessions. Don't let it bite you in the ass.

**Comment by kevr** [comment]
A small workaround to this would be... SpeedSouls could possibly save a cloned copy of the current game save that isn't imported into SpeedSouls, so that users could still reach that save in a case like this. Would this be possible?

Although, I'm not sure how straight-forward this would be, seeing as the save file changes quite often.

I could probably help with this if you guys don't have the time. I'm thinking about polling the save file, and moving diffs of the save file binary over to the "cloned" save file, that way it wouldn't be incredibly expensive to use.

**Comment by Kahmul** [comment]
I added a warning to the readme that the organizer is not meant to be used to first playthroughs and has to be used with care if you do use it for that.
As for your issue, never heard of anyone else have that issue nor have I experienced it myself and wouldn't know what would cause it on the organizer's end. I suggest looking for the issue elsewhere, but feel free to implement something yourself in a separate fork if that is still of importance to you a year later.
